data-reconciliation-app/docs/adr/01-data-ingest.md

```diff
@@ -204,7 +202,7 @@ An E2E data flow is captured [here](../images/data_recon_sample.png) with input
 
 ### 4. K-V Design - Expensive to Reconcile & Report
 
-By only storing the `ReconciledRecord`, we make "updating" of a key for a particular member harder. Addititionally, when we create a report on the reconciled data, members/users will only receive a report on the keys they submitted. By using the `ReconciledRecord`, we will have to check the values for each key to create the report.
+By only storing the `ReconciledRecord`, we made key updation for a particular member harder. Additionally, when we create a report on the reconciled data, members/users will only receive a report on the keys they submitted. By using the `ReconciledRecord`, we will have to check the values for each key to create the report.
```

Comment:
Do we want to explain why it is harder/expensive? 

"Additionally, when we create a report on the reconciled data, members/users will only receive a report on the keys they submitted" - I feel like this is a requirement/irrelevant to this point?

Reply:
Nit: Is updation a word? 

Maybe - "By only storing the `ReconciledRecord`, updating a key for a particular member is more expensive"

Reply:
Acknowledged!  
Updated the ADR